docs/geedocs/5.3.1/answer/3481558.html

```diff
@@ -97,6 +97,35 @@ <h3>Options</h3>
   <pre>
   <code><b>--maxleveloverride </b><i>level</i></code></pre>
   <p><i>Optional</i>. Sets the maximum level for the imagery. Uses the imagery scale in Fusion. Matches the levels in the Fusion UI.</p>
+
+  <h2>geaddtoterrainproject</h2>
```

Comment:
What's the difference between this and gemodifyterrainproject

Reply:
gemodifyterrainproject is more generic and does not requires an asset name to be passed (not mandatory) while geaddtoterrainproject is specific to add a resource terrain to a project. Through gemodifyterrainproject one can change the project without passing necessarily a terrain resource.